2-mandatory/8-journey-planner.js

```diff
@@ -19,7 +19,14 @@
   What's better about each approach?
 */
 
-function journeyPlanner(locations, transportMode) {}
+function journeyPlanner(locations, transportMode) {
+  let availableLocations = [];
+  for (const location in locations) {
```

Comment:
How could you use `Array.filter` and `Array.map` here to solve this problem?

Reply:
We won't be able to use these methods because we have an object as a parameter.

Reply:
Sorry my apologies!